src/app/static/src/app/components/header/FileMenu.vue

```diff
@@ -24,6 +24,35 @@
                 </button>
             </template>
         </modal>
+        <modal id="load-project-name" :open="requestProjectName">
+            <h4 v-translate="'loadFileToProjectHeader'"></h4>
+            <h5 v-translate="'enterProjectName'"></h5>
+            <template v-slot:footer>
+                <div class="container">
+                    <div class="row">
```

Comment:
Sorry didn't notice this before but this is a bit weird - the buttons make sense in the footer, but the actual input should probably be in the modal body? Also the buttons being full width is inconsistent with our modal styling elsewhere, e.g. https://github.com/mrc-ide/hint/blob/master/src/app/static/src/app/components/projects/ProjectHistory.vue#L51

Although in that linked file I notice that the copy project modal is also styled in this idiosyncratic way. We should open a ticket to make that consistent also.

Reply:
Ha, yeah, I made this use the same styling as the copy modal as I figured they should be consistent with each other! But yes, will make this consistent with our other modals too.

Reply:
Added ticket to change the styling in copy modal: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-1791

Reply:
Yeah I figured as much! My fault for not noticing the weird styling on the copy project modal when I reviewed that one! Thanks :)